public/plugin.ts

```diff
@@ -91,6 +103,7 @@ export class notificationsDashboardsPlugin
         title: 'Channels',
         order: 9070,
         workspaceAvailability: WorkspaceAvailability.outsideWorkspace,
+        updater$: this.appStateUpdater,
```

Comment:
What does this do?

Reply:
it provides global data persistence as per -> https://github.com/opensearch-project/OpenSearch-Dashboards/blob/f9965cd29afec8996a0c7b7f5e6001ab41de924e/src/plugins/opensearch_dashboards_utils/docs/global_data_persistence.md?plain=1#L20